### PR TITLE
add webhook function

### PR DESCRIPTION
**Pull request by OlegBondarUA**
#103 

приходить відповідь 
POST /webhook/                 200 OK

але бот перестав реагувати на команду /start

### CODE REVIEW

File: scripts/bot.py [hunk 1]
```diff
@@ -1018,9 +1020,20 @@ def get_update_report(update, context):
...
 
+bot = Bot(token=os.environ['TELEGRAM_TOKEN'])
+updater = Updater(bot=bot, use_context=True)
+dp = updater.dispatcher
+
+
+@csrf_exempt
+def webhook(request):
+    if request.method == 'POST':
+        update = Update.de_json(json.loads(request.body), bot)
```
Comment (SpaceIgor) [inline scripts/bot.py:1031]: Update.de_json(request.body, bot)) try this

File: scripts/bot.py [hunk 2]
```diff
@@ -1049,6 +1062,7 @@ def main():
     # Commands for Users
     # Ordering taxi
     dp.add_handler(CommandHandler("start", start))
+    dp.add_handler(MessageHandler(Filters.text('/start'), start))
```
Comment (SpaceIgor) [inline scripts/bot.py:1065]: why do we need this line???? Check the commands to see if others work

File: scripts/bot.py [hunk 3]
```diff
@@ -1122,7 +1136,12 @@ def main():
     dp.add_handler(MessageHandler(Filters.text('Update report'), get_update_report))
 
     updater.job_queue.run_daily(auto_report_for_driver_and_owner, time=datetime.time(7, 0, 0), days=(1,))
-    updater.start_polling()
+    updater.start_webhook(
+        listen="0.0.0.0",
+        port=8000,
+        url_path=os.environ['TELEGRAM_TOKEN']
+    )
+    updater.bot.set_webhook("https://803f-5-1-12-217.eu.ngrok.io/" + os.environ['TELEGRAM_TOKEN'])
```
Comment (SpaceIgor) [inline scripts/bot.py:1144]: we don't have to show the token

File: auto/settings.py [hunk 4]
```diff
@@ -25,7 +25,13 @@
...
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = bool(os.environ['DEBUG'])
-ALLOWED_HOSTS = os.environ['ALLOWED_HOSTS'].split(' ')
+# ALLOWED_HOSTS = os.environ['ALLOWED_HOSTS'].split(' ')
+ALLOWED_HOSTS = [
+    'localhost',
+    '127.0.0.1',
+    '0.0.0.0',
+    '803f-5-1-12-217.eu.ngrok.io'
+]
```
Comment (SpaceIgor) [inline auto/settings.py:34]: go back to ALLOWED_HOSTS = os.environ['ALLOWED_HOSTS'].split(' ')

**Review by SpaceIgor — COMMENTED** [review]
remove the files .DS_Store and selerybeat.. from the commit

File: scripts/bot.py (hunk 3 above)
Comment (SergeyKutsko) [inline scripts/bot.py:1144]: @OlegBondarUA this webhook is wrong and works only temporary for local please add TELEGRAM_WEB_HOOK_URL env variable

**Pull request closed without merging**